tests/Microsoft.Identity.Test.Common/MsalTestConstants.cs

```diff
@@ -66,7 +66,7 @@ internal static class MsalTestConstants
         public const string AuthorityWindowsNet = "https://" + ProductionPrefCacheEnvironment + "/" + Utid + "/";
         public const string B2CAuthority = "https://login.microsoftonline.in/tfp/tenant/policy/";
         public const string B2CLoginAuthority = "https://sometenantid.b2clogin.com/tfp/sometenantid/policy/";
-        public const string B2CRandomHost = "https://sometenantid.randomhost.com/tfp/sometenantid/policy/";
+        public const string B2CCustomDomain = "https://catsareamazing.com/tfp/catsareamazing/policy/";
```

Comment:
>catsareamazing [](start = 55, length = 14)

:)

Reply:
:sunglasses: 

---
In reply to: [270209120](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1017#discussion_r270209120) [](ancestors = 270209120)